src/adagrams.js

```diff
@@ -1,15 +1,128 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const letterValue = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
```

Comment:
Hi Amandda! Great job completing JS Adagrams! Congrats on finishing your first JavaScript project 🎉 Excellent choice with this data structure, I just want to mention that in JavaScript our keys don't need to be strings. Because of the popularity of JSON, it is frequent to see the keys of object literals as quoted strings. The quotes are required in order to be valid JSON, but they are not required in order to be valid JavaScript. Keys in object literals can be expressed in any of the following three ways:
```javascript
var obj7 = {
  one: 'uno',      // No quotes at all
  'two': 'dos',    // Single-quoted string
  "three": 'tres'  // Double-quoted string
};
```